### DISCUSSION TITLE
Funk Wasserzähler möglich mit CC1101 Modul?

### DISCUSSION THREAD

**Discussion by ElDiablo1985** [jomjol/AI-on-the-edge-device#1923] [Ideas]
Hallo,

mit dem Modul CC1101 und 433Mhz/866Mhz ist es ja möglich Funkzähler auszulesen. 
Ist es möglich das Modul zu integreren sodass man es nur mit dem ESP32 verbindet und man bei der Auswahl von aoed es ggf. auswählen kann um ggf. die Funkzähler auszuwählen ? 

https://de.aliexpress.com/item/32821911314.html

Da wäre der Aufwand mit z.B. Kamera Justieren theoretisch nicht nötig.

Grüße

**Comment by jomjol** [comment]
Das ist in diesem Projekt nicht sinnvoll, denn es scheint mir eine andere Art des Auslesens zu sein.
Klingt nach einem weiteren separaten Projekt.

**Comment by ElDiablo1985** [comment]
Naja, wenn man es unübersichtlich mag dann kann man es natürlich auch per Extra Projekt machen.

Ansonsten könnte man ja per Abfrage am Anfang einfach anfragen ob per Funk oder Kamera und dann kann bei Kamera ggf. das gleiche kommen wie bisher. 

Kann aber gut sein das es beim ESP32 nicht geht wegen Speicher Mangel usw...

Aber Funkzähler nehmen immer mehr zu und deswegen wäre es toll wenn da was kommen würde. 

Ich habe zum Beispiel ein Zähler für Kaltwasser und ein Zähler für Heißwasser.

**Comment by jomjol** [comment]
Naja, der ESP32CAM ist ziemlich beschränkt in der Hardware. Das ist nunmal keine eierlegende Wollmilchsau.

Es gibt viele weitere Wünsche: Datenbankenanbindung (möglichst alle denkbarern :-)), Schalter für Bedienung des Zählers (damit ich im Menü was umschalten kann), externe Kontrolle eines Ventils (zum Abschalten), ... die Liste wäre fast beliebig erweiterbar.

Wir haben uns hier darauf eingestellt, die wesentliche Funktion möglichst gut und einfach umzusetzen: Digitalisierung von analogen Zählern ohne direkten Anschluß.

Trotzdem danke für deine Idee und du hast natürlich Recht, dass zukünftig auch Funktschnittstellen benötigt werden. Bestiimmt findest sich da ja auch ein Mitstreiter. Wenn ich mal so einen Zähler habe, dann vielleicht auch gerne zusammen mit mir.

**Comment by ElDiablo1985** [comment]
Ich kann die Zähler leider nicht ausbauen. Wohne in einer Mietwohnung. 
Sonst hätte ich so ein Zähler mal zugeschickt ^^ 

Grüße

**Reply by caco3** [reply]
@ElDiablo1985  Bist Du dir sicher, dass das Funk-Protokoll opensource und dokumentiert ist? ich gehe davon aus, dass es daran scheitern würde, dass das alles proprietär ist!

Zudem denke ich, dass das eine ganz andere Anwendung ist und wenig redundanz mit dem grosteil dieses Projektes hat 8ausser natürlich das Übermitteln der daten an MQTT, InfluxDB, ...)

Und wie @jomjol schon zeigte, wäre uns so eine Erweiterung gar nicht möglich ohne passende Hardware.
